### PR TITLE
chore: fix typos in staking-script.md

### PR DESCRIPTION
**Pull request by 0xxfu**

### CODE REVIEW

File: x/btcstaking/README.md [hunk 1]
```diff
@@ -5,45 +5,46 @@ providers and BTC delegations under them. This includes:
...
-  - [MsgEditFinalityProvider](#msgeditfinalityprovider)
-  - [MsgCreateBTCDelegation](#msgcreatebtcdelegation)
-  - [MsgAddCovenantSigs](#msgaddcovenantsigs)
-  - [MsgBTCUndelegate](#msgbtcundelegate)
-  - [MsgUpdateParams](#msgupdateparams)
-  - [MsgSelectiveSlashingEvidence](#msgselectiveslashingevidence)
-- [BeginBlocker](#beginblocker)
-- [Events](#events)
-- [Queries](#queries)
+- [BTCStaking](#btcstaking)
```
Comment (SebastianElvis) [inline x/btcstaking/README.md:15]: Could we start from the header 2 rather than header 1?

File: x/btcstaking/README.md [hunk 2]
```diff
@@ -697,7 +698,7 @@ Upon `BeginBlock`, the BTC Staking module will execute the following:
    active BTC delegation, then record the reward distribution w.r.t. the active
    finality providers and active BTC delegations.
 
-The logic is defined at [x/btcstaking/abci.go]((./abci.go)).
+The logic is defined at [x/btcstaking/abci.go](<(./abci.go)>).
```
Comment (SebastianElvis) [inline x/btcstaking/README.md:701]:
```suggestion
The logic is defined at [x/btcstaking/abci.go](./abci.go).
```

File: x/btcstaking/README.md [hunk 3]
```diff
@@ -256,7 +257,7 @@ message BTCUndelegation {
     repeated CovenantAdaptorSignatures covenant_slashing_sigs = 5;
     // covenant_unbonding_sig_list is the list of signatures on the unbonding tx
     // by covenant members
-    // It must be provided after processing undelagate message by Babylon
+    // It must be provided after processing undelegate message by Babylon
```
Comment (SebastianElvis) [inline x/btcstaking/README.md:260]: This means the protobuf file that defines `BTCDelegation` message has the typo as well. Could you please also fix that and recompile by `make proto-gen`?

File: x/btcstaking/README.md (hunk 1 above)
Comment (0xxfu) [reply x/btcstaking/README.md]: done.

File: x/btcstaking/README.md (hunk 2 above)
Comment (0xxfu) [reply x/btcstaking/README.md]: done.

**Review by SebastianElvis — APPROVED** [review]
Thanks!

**Pull request merged**